### PR TITLE
Hooking up the database

### PR DESCRIPTION
**Pull request by kevhuang**
This works locally, not sure about on Heroku yet.

You will need to manually create a database locally named joseki.

### CODE REVIEW

**Comment by dmsakamoto** [comment]
Looks good to me :+1: Glad, I could help with the modularity on the config file. Whoever chose that architecture did create one headache for me :)

**Comment by dmsakamoto** [comment]
:+1:

File: server/db/index.js [hunk 1]
```diff
@@ -1,57 +1,93 @@
...
+   * will return all instances of a ShepherdEvent belonging to that user, and each instance would reference the 
+   * Event the user is a shepherd for.
+   * 
+   * See http://sequelize.readthedocs.org/en/latest/docs/associations/#belongs-to-many-associations
+   *
+   * hasMany connects a source with multiple targets, but the targets can only connect to one source
+   *
+   * See http://sequelize.readthedocs.org/en/latest/docs/associations/#one-to-many-associations
+   */
+
```
Comment (ekong2) [inline server/db/index.js:55]: i think you meant to swap the order. 56 should be shepherds, 57 should be shepherdevents, 58 sheep, 59 sheepevent

Comment (stevenshuhyow) [inline server/db/index.js:55]: just confirmed, its good

**Pull request merged**